release-linux-llvm/Makefile

```
@@ -28,6 +28,19 @@ build_tiflash_release:
 image_tiflash_release:
 	$(DOCKER) build -f dockerfiles/Dockerfile-tiflash-rocky8 -t hub.pingcap.net/tiflash/tiflash-server-rocky8 .
 
+# Build tiflash binaries with next-gen features under directory "tiflash" and "tiflash-columnar"
+build_tiflash_next_gen_release:
```

checkmake (0.3.2)

[warning] Target body for "build_tiflash_next_gen_release" exceeds allowed length of 5 lines (6).

(maxbodylength)